### PR TITLE
Documentation for Octavia Barclamp (SOC-8726)

### PR DESCRIPTION
**Pull request by asettle**
Creating the documentation for Octavia barclamp
and updating existing documentation.

### CODE REVIEW

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -5990,11 +5985,234 @@ monitoring</guimenu>
    </mediaobject>
   </figure>
  </sect1>
+ <sect1 xml:id="sec-depl-ostack-octavia">
+  <title>Deploying octavia</title>
+  <para>
+    &octavia; provides &productname; Load Balancing as a Service (LBaaS). It is
```
Comment (guangyee) [inline xml/depl_nodes.xml]: Shouldn't this be "&productname; provides &octavia;"?

**Comment by jeremy-moffitt** [comment]
Content looks good to me... travis is complaining about the body of some of the commits:
```
Commit ac6c2dac1b:
3: B5 Body message is too short (8<20)
Commit ff5da06538:
3: B5 Body message is too short (14<20)
```

so we need to either amend those or tweak the commit validation ... I'd defer to @asettle @dmpop and @sknorr about the motivation for the commit body length

looks like we also need to rebase this

**Comment by nicolasbock** [comment]
We could squash all the commits into one.

**Comment by jeremy-moffitt** [comment]
> We could squash all the commits into one.

That makes sense to me

**Comment by nicolasbock** [comment]
I squashed the changes and rebased.

**Pull request merged**